### DISCUSSION TITLE
different scales look alike in ar

### DISCUSSION THREAD

**Discussion by heikog** [google/model-viewer#3503] [Q&A]
Hi Guys, another scaling question:
I have on a test site the same mode twice:
(I use blender)
For the first one I scaled down to y=1m and applied scale
the second I scaled up to y=3m and applied scale.
I then used blenders glb exporter. (I am testing Android right now)
I cant see one model being 3x as large as the other - sizes seem a bit arbitrary while testing.
here ist the page:
https://kulturschokolade.org/ar/
Any advice`?
Thanks in advance

**Comment by elalish** [comment]
Can you share screen recordings of the two situations so we can see what you mean? In 3D mode they should look the same, since the camera frames the model. In AR, you'll only get proper apparent scale if you move your phone enough to get it placed on the floor. You can check `ar-status` to see if it's detected your floor yet.
